src/editors/containers/ProblemEditor/components/EditProblemView/ExplanationWidget/index.test.tsx

```diff
@@ -36,19 +16,24 @@ jest.mock('../../../../../sharedComponents/TinyMceWidget', () => ({
   default: () => <div>TinyMceWidget</div>,
 }));
 
-describe('SolutionWidget', () => {
-  const props = {
+const initialState = {
+  problem: {
     settings: { solutionExplanation: 'This is my solution' },
+  },
+  app: {
     learningContextId: 'course+org+run',
     images: {},
     isLibrary: false,
```

Comment:
```suggestion
```
I know you didn't add this line, but once we put the types in you can see it needs to be removed.